shared/studio/components/visualQuerybuilder/querybuilder.module.scss

```diff
@@ -193,6 +194,31 @@
   }
 }
 
+.checkbox {
+  margin-right: 6px;
+  width: 12px;
+  height: 12px;
+  cursor: pointer;
+  &:after {
+    content: " ";
+    display: inline-block;
+    position: relative;
+    width: 12px;
+    height: 12px;
+    margin-right: 4px;
+    background: var(--app-panel-bg);
+    border: 1px solid var(--app-text-colour);
+    border-radius: 2px;
+  }
+  &.checked {
```

Comment:
There's a `:checked` pseudo-class that you use instead of needing to define your own. (https://developer.mozilla.org/en-US/docs/Web/CSS/:checked)